### PR TITLE
Update Blueprint copy

### LINKED ISSUE #53972: [Blueprint] Remove copy created for proof of concept
## Issue Description

Remove the copy created for the proof of concept

* Section title
* Intro copy

### PR DESCRIPTION
**Pull request by adrianduffell**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Fixes #53972.

This PR updates the copy on the Blueprint screen to match the Figma design.

* Removes main title
* Removes intro copy
* Updates section titles
* Replaces a prime marks with apostrophes/quotation marks

Screen Shot 2024-12-23 at 10 28 41 am

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the blueprint feature flag. 
2. Navigate to WooCommerce -> Settings -> Advanced -> Blueprint
3. Compare rendered design to [Figma](https://www.figma.com/design/Y5pUYSJPsGEud1vknUZhi8/Core-onboarding---Profiler?node-id=3791-13011&t=aCCWcj1XqW8zYRBc-4) (attached) 

image

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [x] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 
Update to copy on Blueprint settings screen.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js [hunk 1]
```diff
@@ -124,30 +124,11 @@ const Blueprint = () => {
...
-					to help shape the direction of this feature!
-				</strong>
-			</p>
-			<h3>Import</h3>
+			<h3>Import Blueprint</h3>
 			<p>
 				{ createInterpolateElement(
 					__(
-						"Import your blueprint schema to ease the setup process for your store or allow teams to apply each others' changes. You can import only one Blueprint at the time. Want to know more? Check our <docLink/>",
+						'Import your blueprint schema to ease the setup process for your store or allow teams to apply each others’ changes. You can import only one Blueprint at the time. Want to know more? Check our <docLink/>',
```
Comment (ilyasfoo) [inline plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js]: Nit: In the design, the link is `Check our documentation`, but currently it's just `documentation`

image

**Review by ilyasfoo — COMMENTED** [review]
@adrianduffell  Thanks for addressing! Tests well with a nit comment. 

Also, changelog is missing.

File: plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js (hunk 1 above)
Comment (ilyasfoo) [reply plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js]: Additionally, it seems there is a difference in underline for links in the UI for both documentation links, though I'm unsure what is the cause 🤔 

image

Comment (adrianduffell) [reply plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js]: @ilyasfoo I updated the links in aaa888d. I think maybe we want to have a global class for styling inline documentation links 🤔  Perhaps we should add a follow-up issue to update other links to use the `woocommerce-admin-inline-documentation-link` and put the CSS in a shared location?

File: plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js [hunk 2]
```diff
@@ -165,12 +152,15 @@ const Blueprint = () => {
 			<p className="export-intro">
 				{ createInterpolateElement(
 					__(
-						'Export your blueprint schema. Select the options you want to export, then click on "Export". Want to know more? <docLink/> ',
+						'Export your blueprint schema. Select the options you git want to export, then click on “Export”. Want to know more? <docLink/> ',
```
Comment (ilyasfoo) [inline plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js]: Not intended I assume
```suggestion
						'Export your blueprint schema. Select the options you want to export, then click on “Export”. Want to know more? <docLink/> ',
```

File: plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js (hunk 1 above)
Comment (ilyasfoo) [reply plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js]: > I think maybe we want to have a global class for styling inline documentation links 🤔 Perhaps we should add a follow-up issue to update other links to use the woocommerce-admin-inline-documentation-link and put the CSS in a shared location?

Yeah sounds good, we can standardize all documentation links in a follow up!

**Review by ilyasfoo — APPROVED** [review]
Nice, thanks for addressing! Looks good but I think there's an unintentional change. Pre-approving

File: plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js (hunk 2 above)
Comment (adrianduffell) [reply plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js]: 🤣

File: plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js (hunk 1 above)
Comment (adrianduffell) [reply plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js]: Great, I've added the issue! https://github.com/woocommerce/woocommerce/issues/53980

**Pull request merged**